### ISSUE TITLE
Zoom acquistion of Keybase

### ISSUE THREAD

**Issue by morgzz** [keybase/client#24115]
Anyone else concerned about this?

**Comment by chindraba-work** [comment]
There seems to be a few concerned users, yes. Check out issue  #24105

**Comment by yericua** [comment]
No, I am very happy. Now Zoom users can get xlm wallet. how cool!

**Comment by chindraba-work** [comment]
XLM wallets were available before Keybase added it to their app. Even Atomic Wallet has Stellar XLM in it.

**Comment by yericua** [comment]
Well, it's not in zoom app. having wallet in zoom app is beneficial

**Comment by chindraba-work** [comment]
Nothing in the notice indicated that Zoom was going to integrate the Keybase app, or any of its parts, into the Zoom app. Only that the Keybase brains were going to fix the Zoom security flaws. (Assuming that's even possible, of course.)

**Comment by chindraba-work** [comment]
Merging of the two apps, if it's within the limits of the code base of both, would be a decent outcome. Speculation runs high, however, that the Keybase app, and it's tools, will die a slow death from lack of attention while the brains behind it work on Zoom's security flaws.

**Comment by textileconservation** [comment]
Money talks.

**Comment by yericua** [comment]
with this acquisition, we can see spread e2ee usage to the masses

**Comment by yericua** [comment]
> Money talks.

Its acquisition is to bring secured communication to the masses. Zoom has large userbase, it makes sense for this

**Comment by OneCricketeer** [comment]
Ever heard of Skype?

https://support.skype.com/en/faq/FA31/Does-Skype-use-encryption?

Microsoft actually cares about open source initiatives and moderately, online security (at least evident by investments into FidoU2F)

**Comment by OneCricketeer** [comment]
Also gonna put this here - https://rocket.chat/2020/04/13/alternatives-to-zoom/

**Comment by yericua** [comment]
https://www.owllabs.com/blog/best-meeting-apps
i only use the #1 platform

**Comment by OneCricketeer** [comment]
Lists are biased. It's like only ever reading the first result in Google and not actually comparing it to Bing, Yahoo, DuckDuckGo, StartPage, or any of the others. 

You know what. Whatever. It's not worth it. My grandparents and I will keep using Skype while I use Discord with my gaming friends.

**Comment by OneCricketeer** [comment]
And did you seriously just join Github to comment on this issue? I'm really curious if you know how broad the video conferencing market really is, or even the history of why Zoom was created.

**Comment by OneCricketeer** [comment]
Okay @yericua use your words. What do you have against my comments?

**Comment by yericua** [comment]
> Lists are biased. It's like only ever reading the first result in Google and not actually comparing it to Bing, Yahoo, DuckDuckGo, StartPage, or any of the others.
> 
> You know what. Whatever. It's not worth it. My grandparents and I will keep using Skype while I use Discord with my gaming friends.

sorry @OneCricketeer , you're very wrong. https://zapier.com/blog/best-video-conferencing-apps/ 
zoom is no 1 indeed

**Comment by OneCricketeer** [comment]
It didn't use to be.

Stop reading click bait

**Comment by yericua** [comment]
No, you're outdated, please research properly.

**Comment by chindraba-work** [comment]
Click bait or not aside, and personal opinions ignored as irrelevant, what's left of this conversation? Personal attacks? If y'all want to argue, create a team in Keybase and have a good time. If there is information relavent to the "issue" here, please post what applies.

**Comment by OneCricketeer** [comment]
TBH, mods should be in here locking down the comments. Otherwise, we get loads of issues like OP's or like #24106  which have zero  relevant discussion, as evident by the multiples of votes on the posts. 

I only chimed in that secure platforms clearly already existed for years. The market valuation of Zoom is stupid, and Keybase acquisition largely upsets me even if it does make the platform better. It's just a capitalized land grab because Zoom couldn't hire enough security engineers to start with

**Comment by chindraba-work** [comment]
True that on part 1. I suppose the "mods" are rather busy, however, and can just ignore this, and other, issues. There have been nearly 60 commits to the client repo since the switch, not counting any other repos which might have commits.

The continued activity on the Keybase repos keeps me hopeful for the future of Keybase.

**Comment by lord-aerion** [comment]
> Speculation runs high, however, that the Keybase app, and it's tools, will die a slow death from lack of attention

This is clearly exactly what's happening, judging by the huge number of open issues without even as much as an acknowledgment, and the number of commits that's dwindled to practically zero.

No surprise really, as this is typical of such aquisitions. Remember Yahoo buying the hugely popular Astrid?

Keybase is dead in the water.
